### PR TITLE
Bump version to 3.0.0rc4

### PR DESCRIPTION
**Pull request by BIGWangYuDong**
Add changelog and bump version to 3.0.0rc4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,11 +75,9 @@ Apart from MMDetection, we also released [MMEngine](https://github.com/open-mmla
 
 ## What's New
 
-**v3.0.0rc3** was released in 4/11/2022:
+**v3.0.0rc4** was released in 23/11/2022:
 
-- Support [CrowdDet](configs/crowddet/) and EIoULoss
-- Support training detection models in Detectron2, see [examples](configs/misc)
-- Refactor [Fast R-CNN](configs/fast_rcnn/)
+- Add `projects/` folder, which will be a place for some experimental models/features.
```
Comment (RangiLyu) [inline README.md:80]: Projects folder is for communities to contribute their projects. Also add a link to the example project.

File: docs/en/notes/changelog.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+### New Features
+
+- Add `projects/` folder, which will be a place for some experimental models/features (#9341)
+
+### Bug Fixes
+
+- Fix `pixel_decoder_type` discrimination in MaskFormer Head. (#9176)
+- Fix wrong padding value in cached MixUp (#9259)
+- Rename `utils/typing.py` to `utils/typing_utils.py` to fix `collect_env` error (#9265)
+- Fix resume arg confilct (#9287)
```
Comment (RangiLyu) [inline docs/en/notes/changelog.md:18]:
```suggestion
- Fix resume arg conflict (#9287)
```

File: docs/en/notes/changelog.md [hunk 3]
```diff
@@ -1,5 +1,49 @@
 # Changelog of v3.x
 
+## v3.0.0rc4 (23/11/2022)
+
+### Highlights
+
+- Support [CondInst](https://arxiv.org/abs/2003.05664)
```
Comment (RangiLyu) [inline docs/en/notes/changelog.md:7]: Add link to the config folder

File: README.md [hunk 4]
```diff
@@ -75,11 +75,10 @@ Apart from MMDetection, we also released [MMEngine](https://github.com/open-mmla
 
 ## What's New
 
-**v3.0.0rc3** was released in 4/11/2022:
+**v3.0.0rc4** was released in 23/11/2022:
 
-- Support [CrowdDet](configs/crowddet/) and EIoULoss
-- Support training detection models in Detectron2, see [examples](configs/misc)
-- Refactor [Fast R-CNN](configs/fast_rcnn/)
+- Support [CondInst](https://arxiv.org/abs/2003.05664)
```
Comment (RangiLyu) [inline README.md:80]: Add link to the config folder.

Comment (RangiLyu) [reply README.md:80]: Also, add CondInst to the table in `Overview of Benchmark and Model Zoo`.

File: README.md [hunk 5]
```diff
@@ -75,11 +75,10 @@ Apart from MMDetection, we also released [MMEngine](https://github.com/open-mmla
...
+- Add [`projects/`](./projects/) folder and give an [example](./projects/example_project/README.md) for communities to contribute their projects.
```
Comment (RangiLyu) [inline README.md:81]: sparse inst project

File: README_zh-CN.md [hunk 6]
```diff
@@ -74,11 +74,10 @@ MMDetection 是一个基于 PyTorch 的目标检测开源工具箱。它是 [Ope
...
 ## 最新进展
 
-**v3.0.0rc3** 版本已经在 2022.11.4 发布：
+**v3.0.0rc4** 版本已经在 2022.11.28 发布：
 
-- 支持了 [CrowdDet](configs/crowddet/) 和 EIoULoss
-- 支持训练 Detectron2 中的检测模型并提供了三个[样例](configs/misc)
-- 重构了 [Fast R-CNN](configs/fast_rcnn/)
+- 支持了 [CondInst](https://arxiv.org/abs/2003.05664)
+- 添加了 [`projects/`](./projects/) 文件夹并提供了[示例](./projects/example_project/README.md)，方便社区快速地贡献相关项目
```
Comment (ZwwWayne) [inline README_zh-CN.md:80]: Also indicate sparseinst

**Review by RangiLyu — APPROVED** [review]

**Review by RangiLyu — DISMISSED** [review]

**Comment by BIGWangYuDong** [comment]
Have fixed lint

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**